### PR TITLE
fix: styles in the Picklist component when the prop readOnly is passed

### PR DESCRIPTION
**Pull request by rgah2107**
fix: #1683

## Changes proposed in this PR:
- styles in the Picklist component when the prop readOnly is passed

- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@nexxtway/react-rainbow

### CODE REVIEW

File: src/components/Picklist/styled/input.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ const PickerInput = attachThemeAttrs(styled(StyledInput))`
         box-shadow: none;
         background-color: transparent;
         border: 2px solid transparent;
+        pointer-events: none;
```
Comment (yvmunayev) [inline src/components/Picklist/styled/input.js:46]: The problem with this solution is that it changes the course to the hand that indicates some action and the course should be the text

File: src/components/Input/label/index.js [hunk 2]
```diff
@@ -29,6 +29,7 @@ Label.propTypes = {
     required: PropTypes.bool,
     inputId: PropTypes.string,
     readOnly: PropTypes.bool,
+    disabled: PropTypes.bool,
```
Comment (yvmunayev) [inline src/components/Input/label/index.js:32]: To change the API of a component a previous discussion is necessary, this small change has a big impact, you had to change 5 component which in turn many more components depends on them for this reason it is not feasible. I think the best solution would be to add to the RenderIf where the Label is painted the conditions of disabled and readOnly

**Comment by TahimiLeonBravo** [comment]
> Y
> Screen Shot 2020-07-10 at 4 37 26 AM
> ou have to take into account keyboard navigation

we have a bug when use the keyboard navigation, you can review the bug on the video here
https://drive.google.com/file/d/1FisnzjtXsylVphZBOBUdoaOdrcNz4cSz/view?usp=sharing

**Comment by TahimiLeonBravo** [comment]
@rgah2107 the label should be centered on the Picklist component when `readOnly` and `disabled` are passed at the same time
Screen Shot 2020-07-12 at 5 06 16 PM

**Pull request merged**